lib/rubocop/cop/lint/void.rb

```diff
@@ -126,7 +131,11 @@ def check_void_expression(node)
         def check_nonmutating(node)
           return unless NONMUTATING_METHODS.include?(node.method_name)
 
-          add_offense(node, message: format(NONMUTATING_MSG, method: node.method_name))
+          if METHODS_REPLACABLE_BY_EACH.include?(node.method_name)
+            add_offense(node, message: format(REPLACEABLE_BY_EACH_MSG, method: node.method_name))
+          else
+            add_offense(node, message: format(NONMUTATING_MSG, method: node.method_name))
+          end
```

Comment:
Can you update it as below? Because the new `REPLACEABLE_BY_EACH_MSG` constant can be removed.

```diff
-        NONMUTATING_MSG = 'Method `#%<method>s` used in void context. Did you mean `#%<method>s!`?'
-        REPLACEABLE_BY_EACH_MSG = 'Method `#%<method>s` used in void context. Did you mean `#each`?'
+        NONMUTATING_MSG = 'Method `#%<method>s` used in void context. Did you mean `#%<suggest>s`?'```
```

```ruby
        def check_nonmutating(node)
          method_name = node.method_name
          return unless NONMUTATING_METHODS.include?(method_name)

          suggest = METHODS_REPLACABLE_BY_EACH.include?(method_name) ? 'each' : "#{method_name}!"
          add_offense(node, message: format(NONMUTATING_MSG, method: method_name, suggest: suggest))
        end
```